### PR TITLE
Add Clarifai banner

### PR DESCRIPTION
**Pull request by tot-ra**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,13 @@
-<h1 align="center">
-  <a href="https://www.clarifai.com/"><img alt="Clarifai" title="Clarifai" src="https://upload.wikimedia.org/wikipedia/commons/b/bc/Clarifai_Logo_FC_Web.png"></a>
-</h1>
+![image](https://github.com/user-attachments/assets/623b883b-7fe5-4b95-bbfa-8691f5779af4)
```
Comment (sanjaychelliah) [inline README.md:1]:
```suggestion
<h1 align="center">
  <a href="https://www.clarifai.com/"><img alt="Clarifai" title="Clarifai" src="https://github.com/user-attachments/assets/623b883b-7fe5-4b95-bbfa-8691f5779af4"></a>
</h1>
```

File: README.md [hunk 2]
```diff
@@ -1,14 +1,13 @@
...
 
-<h2 align="center">
-Clarifai Python SDK</a>
+
+<h2>
```
Comment (sanjaychelliah) [inline README.md:4]:
```suggestion
<h2 align="center">
```

File: README.md [hunk 3]
```diff
@@ -1,14 +1,13 @@
...
+Clarifai Python SDK
 </h2>
 
 
 
-<p align="center">
+<p>
```
Comment (sanjaychelliah) [inline README.md:10]:
```suggestion
<p align="center">
```

**Review by sanjaychelliah — COMMENTED** [review]
Thanks for this!

Aligning to the center looks good, WDYT?
Screenshot 2024-10-10 at 8 49 20 PM

**Comment by tot-ra** [comment]
@sanjaychelliah updated

**Review by sanjaychelliah — APPROVED** [review]
👍

**Pull request merged**